### PR TITLE
Task02

### PR DESCRIPTION
**Pull request by ksalhab89**
first commit:
added postgres to build.gradle
modified Car
added jdmc template to CarRepository
added spring.datasource details to application.properties
added the script to create the database for cars and carshop

### CODE REVIEW

File: src/main/resources/SpringOutputAfterStarting [hunk 1]
```diff
@@ -0,0 +1,22 @@
...gradle\caches\modules-2\files-2.1\ch.qos.logback\logback-classic\1.2.11\4741689214e9d1e8408b206506cbe76d1c6a7d60\logback-classic-1.2.11.jar;C:\Users\ksalhab\.gradle\caches\modules-2\files-2.1\org.apache.logging.log4j\log4j-to-slf4j\2.17.2\17dd0fae2747d9a28c67bc9534108823d2376b46\log4j-to-slf4j-2.17.2.jar;C:\Users\ksalhab\.gradle\caches\modules-2\files-2.1\org.slf4j\jul-to-slf4j\1.7.36\ed46d81cef9c412a88caef405b58f93a678ff2ca\jul-to-slf4j-1.7.36.jar;C:\Users\ksalhab\.gradle\caches\modules-2\files-2.1\org.springframework\spring-jcl\5.3.23\3c7eb5fcca67b611065f73ff4325e398f8b051a3\spring-jcl-5.3.23.jar;C:\Users\ksalhab\.gradle\caches\modules-2\files-2.1\ch.qos.logback\logback-core\1.2.11\a01230df5ca5c34540cdaa3ad5efb012f1f1f792\logback-core-1.2.11.jar;C:\Users\ksalhab\.gradle\caches\modules-2\files-2.1\org.apache.logging.log4j\log4j-api\2.17.2\f42d6afa111b4dec5d2aea0fe2197240749a4ea6\log4j-api-2.17.2.jar;C:\Users\ksalhab\.gradle\caches\modules-2\files-2.1\org.checkerframework\checker-qual\3.5.0\2f50520c8abea66fbd8d26e481d3aef5c673b510\checker-qual-3.5.0.jar;C:\Users\ksalhab\.m2\repository\org\jetbrains\annotations\20.1.0\annotations-20.1.0.jar com.ksalhab.caragency.CaragencyApplication
```
Comment (Andriireva) [inline src/main/resources/SpringOutputAfterStarting:1]: it looks like this file was added as mistake, you can delete it from PR

File: src/main/resources/DatabaseTablesCreation.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
+-- Table: public.cars
```
Comment (Andriireva) [inline src/main/resources/DatabaseTablesCreation.txt:1]: it is better to change file extension to .sql to be more consistance

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1 +1,4 @@
-
+spring.datasource.url=jdbc:postgresql://localhost:5432/carAgency
```
Comment (Andriireva) [inline src/main/resources/application.properties:1]: name should be "car_agency" ( it is optional change, feel free to keep it  as it)

File: src/main/resources/DatabaseTablesCreation.txt (hunk 2 above)
Comment (ksalhab89) [reply src/main/resources/DatabaseTablesCreation.txt:1]: will do.

File: src/main/resources/application.properties (hunk 3 above)
Comment (ksalhab89) [reply src/main/resources/application.properties:1]: will do

File: src/main/resources/SpringOutputAfterStarting (hunk 1 above)
Comment (ksalhab89) [reply src/main/resources/SpringOutputAfterStarting:1]: i added it to show the output, will be deleted.

**Pull request merged**